YeomanTemplates/generator-choAdmin-boilerplate/generators/app/templates/package.json

```
@@ -0,0 +1,72 @@
+{
+  "name": "@canadahelps/<%= packageName %>",
+  "version": "<%= apiVersion %>",
+  "description": "<description>",
+  "main": "src/index<.ts|.js|.tsx>",
+  "scripts": {
+    "prepublishOnly": "npm test",
+    "prepare": "run-s clean transpile",
+    "clean": "rimraf lib",
+    "copy-style": "cpx \"src/**/*.{css,scss}\" lib",
+    "transpile": "run-p transpile:babel transpile:declarations copy-style",
+    "transpile:babel": "babel src --out-dir lib --extensions \".ts,.tsx,.js,.jsx\"",
+    "transpile:declarations": "tsc --emitDeclarationOnly",
+    "build-storybook": "build-storybook",
+    "build:dev": "webpack --env.mode development",
+    "build": "webpack --env.mode production --display-error-details",
+    "deploy:dev": "npm run build:dev",
+    "deploy": "npm run build",
+    "storybook": "start-storybook -s ../../dist -p 9009",
```

Please add linting scripts
```suggestion
    "lint": "eslint \"src/**\" --fix",
    "lint:ci": "eslint \"src/**\"",
    "storybook": "start-storybook -s ../../dist -p 9009",
```